src/main/java/net/sf/jabref/gui/entryeditor/PdfCommentsTab.java

```diff
@@ -66,8 +66,9 @@ private void setUpPdfCommentsTab() {
             DefaultListModel<String> listModel = new DefaultListModel<>();
             commentList.setModel(listModel);
             PdfCommentImporter commentImporter = new PdfCommentImporter();
-            HashMap<String, String> importedNotes = commentImporter.importNotes(field.get());
-            for (String note : importedNotes.values()) listModel.addElement(note);
+            String pdfPath = field.get().replaceFirst(".*?:", System.getProperty("file.separator")).replaceAll(":PDF", "");
```

Comment:
But still I think we should take care about that. The simplest way would be to create a new pdf tab for every attached pdf file or a drop down to switch files. I will have a discussion with @ayanai1  and @mairdl  on that.